operator [N] [CI] multicluster-global-hub-operator (0.5.0)

morvencao opened the pull request:
Signed-off-by: morvencao <lcao@redhat.com>

Thanks for submitting your Operator. Please check the below list before you create your Pull Request.

### New Submissions

* [x] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [x] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [x] Have you tested your Operator with all Custom Resource Definitions?
* [x] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [x] Have you considered whether you want to use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [x] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Your submission should not

* [x] Modify more than one operator
* [x] Modify an Operator you don't own
* [x] Rename an operator - please remove and add with a different name instead
* [x] Modify any files outside the above mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [x] Description of the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

J0zi commented:
@morvencao please put your CSV file content to https://operatorhub.io/preview. There are missing fields on the right panel, all under `spec` key like `spec.containerImage` and so. Another point. Please extend description by couple more sentences. There are also issues with `csv.Spec.Icon`.

morvencao commented:
thanks @J0zi for quick response.
I updated the `containerImage` and other fields, added more clear description about prerequisites and usage in the CSV.
For the icon, we haven't get one, can I use default icon mentioned [here](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon)?

J0zi commented:
/test all

J0zi commented:
@morvencao it looks great, just `annotations.createdAt` is missing.

morvencao commented:
@J0zi I'm not sure how to generate `annotations.createdAt`, it should be automatically generated when running `operator-sdk generate bundle`, right? but I found it is not generated.

morvencao commented:
Updated PR status to "Working In Progress" because we need to configure CI to build operator and operand images used in the bundle.

mvalarh commented:
/test all

framework-automation commented:
/merge possible

framework-automation commented:
/merge possible

mvalarh commented:
/retest

framework-automation commented:
/merge possible

framework-automation commented:
/merge possible

framework-automation commented:
/merge possible

dockerymick reviewed (commented):
Please excuse the tardiness of my review, there were other issues that took longer than expected. thank you for receiving my feedback!
